src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/JudicialDecisionNotificationHandler.java

```diff
@@ -312,9 +341,19 @@ private void applicationDirectionOrder(CaseData caseData) {
     }
 
     private void judgeApprovedOrderApplicationCloak(CaseData caseData) {
-        sendNotificationForJudicialDecision(caseData,
-            caseData.getGeneralAppApplnSolicitor().getEmail(),
-            notificationProperties.getJudgeForApprovedCaseApplicantEmailTemplate());
+        if (useDamageTemplate(caseData)) {
+            sendNotificationForJudicialDecision(caseData,
+                                                caseData.getGeneralAppApplnSolicitor().getEmail(),
```

Comment:
caseData.getGeneralAppApplnSolicitor().getEmail(),
can be fetched once before 'if' and the referred to in all places.

Reply:
Good point - I update it as well in other part of code with the same situation.